lib/builder-webpack4/src/preview/virtualModuleModernEntry.js.handlebars

```diff
@@ -1,39 +1,47 @@
 import fetch from 'unfetch';
+import global from 'global';
 
-import { composeConfigs, PreviewWeb } from '@storybook/preview-web';    
-import { ClientApi } from '@storybook/client-api';    
+import { composeConfigs, PreviewWeb } from '@storybook/preview-web';
+import { ClientApi } from '@storybook/client-api';
 import { addons } from '@storybook/addons';
-import createChannel from '@storybook/channel-postmessage';
+import createPostMessageChannel from '@storybook/channel-postmessage';
+import createWebSocketChannel from '@storybook/channel-websocket';
 
 import { importFn } from './{{storiesFilename}}';
 
+const { SERVER_CHANNEL_URL } = global;
+
 const getProjectAnnotations = () =>
-  composeConfigs([
-    {{#each configs}}
-    require('{{this}}'),
-    {{/each}}
-  ]);
+composeConfigs([
```

Comment:
@tmeasday is this indentation intentional?

Reply:
Ughh, no, sorry not sure what happened.